src/main/java/org/folio/circulation/domain/RequestRepresentation.java

```diff
@@ -53,6 +54,11 @@ private static void addStoredItemProperties(JsonObject request, Item item) {
     write(itemSummary, "title", item.getTitle());
     write(itemSummary, "barcode", item.getBarcode());
 
+    String callNumber = item.getCallNumber();
+    write(itemSummary, "callNumber", StringUtils.isBlank(callNumber)
```

Comment:
Please see my other comment about whether it is appropriate to save the `call number`

Without a corresponding change to the schema in mod-circulation-storage this change would break all attempts to create or update requests, due to it being an unexpected additional property.

Reply:
Rollback changes: 5d81e11 / Add callNumber to request representation